test/Kestrel.FunctionalTests/TestHelpers/TestServer.cs

```diff
@@ -77,6 +80,19 @@ public TestServer(RequestDelegate app, TestServiceContext context, ListenOptions
             _host.Start();
         }
 
+        public static void RemoveDevCert(IServiceCollection services)
+        {
+            // KestrelServerOptionsSetup would scan all system certificates on every test server creation
+            // making test runs very slow
+            foreach (var descriptor in services.ToArray())
```

Comment:
/cc @javiercn

Reply:
I guess this is ok if you guys are not using Configuration to setup kestrel for the tests.